examples/ssd_mobilenet/MobileNet.js

```diff
@@ -0,0 +1,215 @@
+let nnPolyfill, nnNative;
+if (navigator.ml.isPolyfill) {
+  nnNative = null;
+  nnPolyfill = navigator.ml.getNeuralNetworkContext();
+} else {
+  nnNative = navigator.ml.getNeuralNetworkContext();
+  nnPolyfill = navigator.ml_polyfill.getNeuralNetworkContext();
+}
```

Comment:
Please remove above code as they are in https://github.com/intel/webml-polyfill/blob/master/examples/util/base.js